superset-frontend/src/views/Store.tsx

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
The pattern so far has been to only capitalize file names if they include a react component. I'm also not seeing any `jsx` in this file, so `store.ts` would be a better name.

Reply:
done

Reply:
@nytai If you can do the merge I appreciate it. I can't do it :-D